lib/static/new-ui/app/App.tsx

```diff
@@ -23,8 +22,7 @@ export function App(): ReactNode {
             element: <SuitesPage/>,
             children: [<Route key={'suite'} path=':suiteId' element= {<SuitesPage/>} />]
         },
-        {title: 'Visual Checks', url: '/visual-checks', icon: Eye, element: <VisualChecksPage/>},
-        {title: 'Info', url: '/info', icon: CircleInfo, element: <InfoPage/>}
```

Comment:
Why did you remove it?

Reply:
I plan to implement info as a panel instead of a page. But right now it's empty so there's no point in misleading users, i will add it back when it'll be ready.